docs/package_templates/cmake_package/all/conanfile.py

```diff
@@ -77,9 +73,12 @@ def validate(self):
         # validate the minimum cpp standard supported. For C++ projects only
         if self.info.settings.compiler.cppstd:
             check_min_cppstd(self, self._minimum_cpp_standard)
-        minimum_version = self._compilers_minimum_version.get(str(self.info.settings.compiler), False)
-        if minimum_version and Version(self.info.settings.compiler.version) < minimum_version:
-            raise ConanInvalidConfiguration(f"{self.ref} requires C++{self._minimum_cpp_standard}, which your compiler does not support.")
+        if is_msvc(self):
+            check_min_vs(self, 191)
+        else:
```

Comment:
```suggestion
        check_min_vs(self, 191)
        if not is_msvc(self):
```
There is an advantage, `check_min_vs` only triggers its check it we are running msvc, internally, it checks it.